SecondHomework/Logistic/Logger.cs

```diff
@@ -15,5 +15,10 @@ public void CheckForIdToRemove()
         Console.WriteLine("Enter please id of the parcel: ");
     }
 
+    public void AskToGetFilteredListOfParcelsByWeight()
+    {
+        Console.WriteLine("Would you like to get a list of parcels filtered by weight?");
```

Comment:
How can you refactor this class to have only one method `void PrintMessage(string message)`?